test/mocks.ts

```diff
@@ -26,4 +26,35 @@ module Mocks {
       return true;
     }
   }
+
+  export class Scale<D, R> extends Plottable.Scale<D, R> {
```

Comment:
In this case though, the mock isn't a "mock" of the parent class -- it's a subclass, which puts it in the same position of the inheritance tree as `Scales.Linear`. In this case, it can't give any more faith that tests based on it will be more representative of the general case than `Scales.Linear` would.

If the behavior of your code should be independent of the type of `Scale` in the `Axis`, it should work regardless of the type of `Scale` used in tests right?